p2panda-rs/src/operation/error.rs

```diff
@@ -10,6 +10,10 @@ pub enum OperationError {
     #[error("operation fields can not be empty")]
     EmptyFields,
 
+    /// Invalid attempt to create a delete operation with fields
+    #[error("DELETE operation must not have fields")]
```

Comment:
"delete" can be lowercase

Reply:
Commented on this in the chat, but I like the uppercase writing: DELETE has a special meaning, it represents the (enum) type of the operation, not a verb.

Reply:
Which btw, is not very consistent in this PR (hihi)

Reply:
I'm quite attached to the all caps version....